symmetric-client/src/main/java/org/jumpmind/symmetric/AbstractCommandLauncher.java

```diff
@@ -207,7 +207,7 @@ protected void configurePropertiesFile(CommandLine line) throws ParseException {
                         line.getOptionValue(OPTION_PROPERTIES_FILE));
             }
         } else if (line.hasOption(OPTION_ENGINE)) {
-            propertiesFile = PropertiesUtil.findPropertiesFileForEngineWithName(line.getOptionValue(OPTION_ENGINE));
+            propertiesFile = PropertiesUtil.findPropertiesFileForEngineWithName(line.getOptionValue(OPTION_ENGINE), null);
             if (propertiesFile == null || (propertiesFile != null && !propertiesFile.exists())) {
```

Comment:
This line could remove the null argument and use the old API.  See my comment on preserving the old API.